components/story-element.js

```diff
@@ -71,6 +71,12 @@ function StoryElementTable({element}) {
   return React.createElement(Table, { id: element.id, data: element.data, hasHeader: element.metadata['has-header']});
 }
 
+function StoryElementFile({element}) {
+  return React.createElement("div", {className: "attachment", key: element.id}, 
```

Comment:
Maybe <div> wrapper is not necessary since StoryElementBase component is already generating wrapper div with proper class names. Maybe you can use React.Fragment

Reply:
I have tried implementing the same, but it doesn't work.